UtopiaCity/Helpers/Automapper/AutoMapperProfile.cs

```diff
@@ -0,0 +1,33 @@
+﻿using AutoMapper;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Threading.Tasks;
+
+namespace UtopiaCity.Helpers.Automapper
+{
+    public class AutoMapperProfile : Profile
+    {
+        public AutoMapperProfile()
+        {
+            CreateMap<UtopiaCity.Models.Life.WeatherReport.Data, UtopiaCity.Models.Airport.WeatherReport>()
```

Comment:
Actually wanted to show this today, nice

Reply:
it would be glad to hear-how to use third-party APIs in our project or how to integrate it !

Reply:
> it would be glad to hear-how to use third-party APIs in our project or how to integrate it !

fetch data and map:
var data = await WeatherReportApi.Get30DayReportAsync();
var reports = _mapper.Map<UtopiaCity.Models.Airport.WeatherReport>(data);
or
var data = await WeatherReportApi.GetReportClosestToDateAsync(date);
var reports = _mapper.Map<List<UtopiaCity.Models.Airport.WeatherReport>>(data);